src/MassTransit.RabbitMqTransport/RabbitMQRoutingkeyFormatter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public class RabbitMqRoutingkeyFormatter: IRoutingkeyFormatter
+    {
+        public string createRoutingkeyForType(Type messageType)
+        {
+            return "";
```

Comment:
Should this return a null instead? Do you know what happens to the configuration?